WallPaper_Project1/WallPaper_Project1/Scenes/PersonalScreen/PersonalViewController.swift

```diff
@@ -8,9 +8,102 @@
+    @IBOutlet private weak var buttonViewContainer: UIView!
+    @IBOutlet private weak var favoriteButton: UIButton!
+    @IBOutlet private weak var collectionView: UICollectionView!
+    private var isPhoto = true
+    private var images = [Image]()
+    private var videos = [Videos]()
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        configView()
+        configCollectionView()
     }
 
+    private func configView() {
+        buttonViewContainer.layer.cornerRadius = 12
+        downloadButton.setTitleColor(.white, for: .normal)
+    }
+
+    private func configCollectionView() {
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(nibName: ImageCollectionViewCell.self)
+        collectionView.register(nibName: VideoCollectionViewCell.self)
+    }
+    
+    private func showPopUp(notice: String) {
+        let popUpView = PopUpViewController(nibName: "PopUpViewController", bundle: nil)
+        popUpView.bindData(notice: notice)
+        addChild(popUpView)
+        view.addSubview(popUpView.view)
+    }
+
+    @IBAction func downloadButtonTapped(_ sender: Any) {
+        downloadButton.setTitleColor(.white, for: .normal)
+        favoriteButton.setTitleColor(.gray, for: .normal)
+        DispatchQueue.main.async { [unowned self] in
+            collectionView.reloadData()
+        }
+    }
+
+    @IBAction func favoriteButtonTapped(_ sender: Any) {
+        downloadButton.setTitleColor(.gray, for: .normal)
+        favoriteButton.setTitleColor(.white, for: .normal)
+        DispatchQueue.main.async { [unowned self] in
+            collectionView.reloadData()
+        }
+    }
+}
+
+extension PersonalViewController: UICollectionViewDataSource {
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+//       update after
+        return 15
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        if isPhoto {
+            guard let cell: ImageCollectionViewCell =
+                    collectionView.dequeueReusableCell(forIndexPath: indexPath) else {
+                showPopUp(notice: "Could not dequeue cell with identifier ")
+                return UICollectionViewCell()
+            }
+            return cell
+        } else {
+            guard let cell: VideoCollectionViewCell =
+                    collectionView.dequeueReusableCell(forIndexPath: indexPath) else {
+                showPopUp(notice: "Could not dequeue cell with identifier ")
+                return UICollectionViewCell()
+            }
+            return cell
+        }
+    }
+}
+
+extension PersonalViewController: UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        let detailScreen = DetailViewController(nibName: "DetailViewController", bundle: nil)
```

Comment:
define `identifier` ở  DetailViewController e
```suggestion
        let detailScreen = DetailViewController(nibName: DetailViewController.identifier, bundle: nil)
```